internal/conduit/version.go

```diff
@@ -0,0 +1,107 @@
+package conduit
+
+import (
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+type Flags struct {
+	args *Args
+}
+
+type Args struct {
+	PipelineFile   string
+	ConnectorsPath string
+	DBPath         string
+	ProcessorsPath string
+}
+
+func NewFlags(fns ...func(*Args)) *Flags {
+	var args Args
+	for _, fn := range fns {
+		fn(&args)
+	}
+	return &Flags{args: &args}
+}
+
+func (f *Flags) ForVersion(ver string) []string {
+	constraints := map[string]string{
+		"v011": "< 0.12.0",
```

Comment:
I think these will be simpler defined using the tilde operator, e.g. `~0.11` will be the equivalent of `>= 0.11, < 0.12` and then the equivalent for 0.12 as well


See https://github.com/Masterminds/semver?tab=readme-ov-file#tilde-range-comparisons-patch